specs/brand-and-query.spec.ts

```diff
@@ -1,40 +1,44 @@
-describe('InstantSearch - Search on specific brand and query filtering', () => {
-  it('navigates to the e-commerce demo', async () => {
-    await browser.url('examples/e-commerce/');
-  });
+export function brandAndQuery(flavor: string) {
```

Comment:
Type should be `typeof flavors[number]`, same for the other factories.

Reply:
I had that first but the `flavor` file needs to be cjs for it to work in the config, and then I'm either writing the flavours twice without validation, or have it as a string (what I preferred in the end)

Reply:
Ah 😞 Can't we use `.mjs` configs with WebdriverIO?

Reply:
with newer versions we can do it in typescript I think, I'll look if possible to upgrade once migrated in monorepo